src/ScillaContractsInfoUpdater.ts

```diff
@@ -20,17 +20,19 @@ type ContractMapByName = Record<ContractName, ContractInfo>;
 export type ScillaContracts = ContractMapByName;
 type ContractMapByPath = Record<ContractPath, ContractInfo>;
 
-export const updateContractsInfo = () => {
+export const updateContractsInfo = (fromDir? : string) => {
   let contractsInfo: ContractMapByName = {};
-  const files = glob.sync("contracts/**/*.scilla");
+  const rawFiles = glob.sync(fromDir ? path.join(fromDir, "/**/*.scilla") : "**/*.scilla");
+  const nodeModules = fromDir ? path.join(fromDir, "node_modules/") : "node_modules/";
+  const files = rawFiles.filter( (f: string) => { console.log(`F= ${f}`); return !f.startsWith( nodeModules ) } )
```

Comment:
I'll submit a PR to remove the debug printf()....

Reply:
Thanks.